flujo/cli/main.py

```
@@ -417,7 +424,22 @@ def dev_health_check(
-    console = Console()
+    try:
+        from rich.console import Console as _Console
+
+        console = _Console()
+    except ModuleNotFoundError:
+        console = None
+    # Ensure console.print works even without Rich
+    if console is None:
+
+        class _PlainConsole:
+            def print(self, msg: object, *args: object, **kwargs: object) -> None:
```

Ruff (0.12.2)

Unused method argument: `args`

(ARG002)